tests/validation/tests/v3_api/test_create_ha.py

```diff
@@ -43,41 +48,67 @@
 export_cmd = "export KUBECONFIG=" + kubeconfig_path
 
 
-def test_create_ha(precheck_certificate_options):
+def test_install_rancher_ha(precheck_certificate_options):
     cm_install = True
-
     if "byo-" in RANCHER_HA_CERT_OPTION:
         cm_install = False
+    print("The hostname is: {}".format(RANCHER_HA_HOSTNAME))
+
+    # prepare an RKE cluster and other resources
+    # if no kubeconfig file is provided
+    if RANCHER_HA_KUBECONFIG is None:
+        nodes = create_resources()
+        rke_config = create_rke_cluster_config(nodes)
```

Comment:
This is same as the global variable `rke_config` --> `from .test_rke_cluster_provisioning import rke_config` . Do we have any chances that it will be overridden here?

Reply:
the variable is renamed to `config_path` which makes more sense here (`create_rke_cluster_config(nodes)` returns the path to the filled config)